sqrl-cli/src/main/java/com/datasqrl/cli/output/TestOutputManager.java

```diff
@@ -31,30 +32,31 @@ public class TestOutputManager implements AutoCloseable {
 
   private final Path rootDir;
 
-  private PrintStream originalOut;
-  private PrintStream originalErr;
+  private final PrintStream originalOut = System.out;
+  private final PrintStream originalErr = System.err;
   private PrintStream logStream;
+  private ErrorCapturingStream errorCapturingStream;
 
-  public void redirectStd() {
-    originalOut = System.out;
-    originalErr = System.err;
+  @SneakyThrows
+  public void init() {
```

Comment:
I don't see any reason for this `init()`. We instantiate this object in `TestCmd` and call `init()` right after, so lets just move the `init()` logic to the ctor. Then `getCapturedErrors` can be simplified to a lombok getter as well.